.github/workflows/fix-code-style.yml

```diff
@@ -23,16 +23,11 @@ jobs:
           tools: composer:v2
 
       - name: Install dependencies
-        run: |
-          composer remove phpro/grumphp vimeo/psalm --no-update --dev
-          composer update --prefer-dist --no-progress
+        run: composer update --prefer-dist --no-progress
 
       - run: composer fix-style
         continue-on-error: true
 
-      # Revert manual modifications so they don't get commited 💥
-      - run: git checkout -- composer.json
-
```

Comment:
Didn't see the reason why we've to manually fiddle with packages which in turn requires to manually exclude `composer.json` from commiting etc. => "simplified" is the word I was looking for